src/generators/css-filter.ts

```diff
@@ -63,7 +77,9 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        // If user has the chrome issue the colors should be the other way around as of the rootcolors will affect the whole background color of the page
+        const rootColors = hasChromiumIssue501582(config) ? [0, 0, 0] : [255, 255, 255];
```

Comment:
Config has no relation to Chromium issue.

Reply:
It's required to check if the config is light or dark don't want to make more in this function

Reply:
`hasChromiumIssue501582` should only say, is there Chromium issue or not. But it's great you don't want a mess in `cssFilterStyleheetTemplate`, it is already messy. You can create a `getDefaultRootColor(config)` function.

Reply:
Refresh for the newest commit :)